examples/google/example.go

```diff
@@ -94,53 +99,47 @@ func login(outputDir string, alg jwa.KeyAlgorithm, signGQ bool) error {
 	if err := json.Indent(&prettyJSON, pktJson, "", "  "); err != nil {
 		return err
 	}
-
 	fmt.Println(prettyJSON.String())
 
 	// Save our signer and pktoken by writing them to a file
-	return signer.WriteToFile()
+	saveLogin(outputDir, signer.(*ecdsa.PrivateKey), pktJson)
+
+	return nil
 }
 
-func googleSign(message string, outputDir string, alg jwa.KeyAlgorithm, signGq bool) error {
-	signer, err := pktoken.LoadFromFile(outputDir, alg.String(), signGq, nil)
+func sign(message string, outputDir string, alg jwa.KeyAlgorithm, signGq bool) error {
+	signer, pkt, err := loadLogin(outputDir)
 	if err != nil {
 		return fmt.Errorf("failed to load client state: %w", err)
 	}
 
-	msgHash := sha256.New()
-	_, err = msgHash.Write([]byte(message))
-	if err != nil {
-		return err
-	}
-	msgHashSum := msgHash.Sum(nil)
-
-	rawSigma, err := signer.Pksk.Sign(rand.Reader, msgHashSum, crypto.SHA256)
+	msgHashSum := sha3.Sum256([]byte(message))
+	rawSigma, err := signer.Sign(rand.Reader, msgHashSum[:], crypto.SHA256)
```

Comment:
This is my silly variable name, since you are touching this code, can you erase my naming sin here and change this to `sig`